sheduling/src/utils/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const host = "http://localhost:3000/login";
+const API = {
+  login: (phone, pass, success) => {
+    axios.post(host, { phone: phone, password: pass }).then(res => {
+      success(res);
```

Comment:
You'll have to create an action creator that returns an action that accepts the token as the payload so that you can dispatch it to the store.